mbed_project/mbed_program.py

```diff
@@ -54,14 +61,23 @@ def from_remote_url(cls, url: str, dst_path: Path) -> "MbedProgram":
                 "to an empty directory."
             )
 
-        logger.info(f"Cloning Mbed program from URL {url}")
+        print(f"Cloning Mbed program from URL '{url}'.")
```

Comment:
should it actually be `print` and not a call to the logger?

Reply:
Not sure. I want these messages to appear regardless of log level. Which is why I went with `print` here.

Reply:
the problem is that if a person redirects logs to a file, these messages would not appear unless I am mistaken.

Reply:
I was trying to make a distinction between user facing messages and developer facing log messages if that makes sense. That's why I left the other logger calls in there. Maybe it's not worth it and I should send everything to the logger? `print` has a `file` arg which defaults to `stdout`, but anyone would have to set that specifically if they were redirecting to a file. I'm wondering if we should set the default log level to "info" or something if we go the logger route? I think it's important these messages are shown to the user in the "default" case without having to pass verbosity flags.

Reply:
fe70a74